src/helpers/constants.js

```diff
@@ -16,3 +16,14 @@ export const codes = {
   DOWN: 40,
   TAB: 9,
 }
+
+export const valid_icons = {
```

Comment:
@MayUWish nit pick: please use all capital letters for naming constants and enums. In this case it would be `VALID_ICONS`

Reply:
Hi @ChirashiSushi  fixed! Thank you!